pkgs/test_api/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.2.5
+
+* Expose the  `Metadata`, `PlatformSelector`, `Runtime`, and `SuitePlatform`
```

Comment:
Do you want to take a pass through other code using this and see if we can get rid of any existing `src/` imports from the other packages? Or should we lump that in with the other cleanup later?

Reply:
I just did that for `test_core` for a few imports, but ya I think we can handle that for other packages later. We could probably search all the packages we pull internally the easiest?